index.d.ts

```diff
@@ -0,0 +1,491 @@
+import drop from "./lib/drop";
+import either from "./lib/either";
+import evolve from "./lib/evolve";
+import fill from "./lib/fill";
+
+declare let nanoutils: Nanoutils.Static
+
+declare namespace Nanoutils {
+  // call
+  interface CallFunction<T1, T2> {
+    (): CallFunction<T1, T2>
+    (...array: T1[]): T2
+  }
+
+  // compact
+  type Falsy = false | 0 | NaN | '' | null | undefined
```

Comment:
`NaN` [cannot be used in union type](https://github.com/Microsoft/TypeScript/issues/15135#issuecomment-296850977)